threesixty/models.py

```diff
@@ -16,6 +16,7 @@ class Survey(models.Model):
     employee_gender = models.CharField(_('employee gender'), max_length=6, choices=GENDERS, default='other')
     employee_email = models.EmailField(_('employee email'))
     manager_email = models.EmailField(_('manager email'))
+    participant_can_skip = models.BooleanField('participant can skip', default=True)
```

Comment:
Please add a small help text, that explains something like: `This option allows a participant to skip single questions.`

Reply:
I would also default this to `False`